### PR TITLE
G-retriever (GNN+LLM) example

### PR DESCRIPTION
**Pull request by puririshi98**
```
Hit: 0.6205
Precision: 0.4565
Recall: 0.5514
F1: 0.4647
Test Acc 0.6205412054120542
E2E time (e2e_time) = 2155.82
E2E time minus Prep Time = 2110.29
```

### CODE REVIEW

**Comment by puririshi98** [comment]
most recent commit just cleans up some old debugging code i had included. My keyboard glitched while inputting the commit message hence the long string of random keys

File: examples/LLM_plus_GNN.py [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+                            pin_memory=True, shuffle=False)
+    test_loader = DataLoader(test_dataset, batch_size=4, drop_last=False,
+                             pin_memory=True, shuffle=False)
+
+    # Step 2: Build Model
+    model = GAT_LLAMA("meta-llama/Llama-2-7b-chat-hf")
+
+    # Step 3 Set Optimizer
+    params = [p for _, p in model.named_parameters() if p.requires_grad]
+    lr = 1e-5
```
Comment (Kh4L) [inline examples/LLM_plus_GNN.py:359]: How about we make some of the hyperparams as script args: lr, epochs, grad_steps etc

**Review by Kh4L — APPROVED** [review]
LGTM!
It would be nice to rename the script `gretriever.py` and move it to a `examples/llm_plus_gnn` dir

**Comment by puririshi98** [comment]
closing in light of new PR
https://github.com/pyg-team/pytorch_geometric/pull/9154/

**Pull request closed without merging**